### ISSUE TITLE
Replace `*` value with other more bash friendly value

### ISSUE THREAD

**Issue by nflaig** [ChainSafe/lodestar#5963]
### Problem description

`*` is not very friendly to use in bash, it has to be quoted or else it will not be handled as a string but an operator instead, i.e. it will list all files of the directory.

There are currently a few CLI flags that accept `*` as a value
- `--rest.namespace`
- `--rest.cors`
- `--keymanager.cors`

See [Command Line Reference](https://chainsafe.github.io/lodestar/reference/cli/) for an overview of all non-hidden flags.

### Solution description

Replace `*` value with other more bash friendly value, for example the word `all`. 

It has to be considered what value to use for cors as `*` has a special meaning in this context.

**Comment by clionachee** [comment]
Hi, may I know specifically which file or all the files? Is this issue still needing to be resolved?

**Comment by nflaig** [comment]
Hey, there are multiple cases but likely just wanna change this for `--rest.namespace` (see related PR https://github.com/ChainSafe/lodestar/pull/2809), and make sure it's backward compatible, i.e. `*` should still be handled if passed by the user.

https://github.com/ChainSafe/lodestar/blob/c7c99412c3bbe9a550811db7b812f5e6fc741357/packages/cli/src/options/beaconNodeOptions/api.ts#L4

It is probably also the only place where we would wanna change this, as for `.cors` CLI flags, the `*` has special meaning, and we can't change it to something like `"all"` there as it might be confusing.

**Comment by clionachee** [comment]
@nflaig if it's still up, I'm interested. Thanks a bunch!

**Comment by clionachee** [comment]
Thanks. I suppose if it's ok, I'll fork the Repo, make the changes in that file, replace the value & commit changes if no problem?

**Comment by nflaig** [comment]
Sure, go ahead, you can open a PR here after you are done with your changes. Happy to review :)

**Comment by clionachee** [comment]
Created PR #6459 thanks w/ love.

**Comment by philknows** [comment]
#7011 target

**Comment by nflaig** [comment]
> [011](https://github.com/ChainSafe/lodestar/issues/7011) target

we should add this in a backward compatible way

**Comment by eniko1556** [comment]
Can we have proper quoting instead of replacing it? ex: ```--rest.namespace '*'``` and so on.

**Comment by nflaig** [comment]
> Can we have proper quoting instead of replacing it? ex: `--rest.namespace '*'` and so on.

Yes, you have to quote it otherwise it won't work, that's why we wanna replace it with a more bash friendly value.

**Comment by guha-rahul** [comment]
@nflaig I did a pr, can you review it?

**Comment by nflaig** [comment]
> [@nflaig](https://github.com/nflaig) I did a pr, can you review it?

Make sure to address bot review comments or close them if they don't make sense, also check the CI failures and fix any lint / type issues.

**Comment by guha-rahul** [comment]
> > [@nflaig](https://github.com/nflaig) I did a pr, can you review it?
> 
> Make sure to address bot review comments or close them if they don't make sense, also check the CI failures and fix any lint / type issues.

done

**Comment by guha-rahul** [comment]
> > [@nflaig](https://github.com/nflaig) I did a pr, can you review it?
> 
> Make sure to address bot review comments or close them if they don't make sense, also check the CI failures and fix any lint / type issues.

can you run the ci/cd again. I was getting no errors when ignore check was deleted. kinda weird.
